### ISSUE TITLE
After clicking eqMac.dmg nothing happens

### ISSUE THREAD

**Issue by smolendawid** [bitgapp/eqMac#24]
After clicking eqMac.dmg nothing happens. Just nothing, no process in Activity Monitor, no window.

MacOS Catalina 10.15.3

**Comment by nodeful** [comment]
That's weird it's supposed to open a Window where you drag eqMac.app into your /Applications folder. Do you see the disk image attached here:
![image](https://user-images.githubusercontent.com/8472525/76068048-55ece180-5f88-11ea-94c0-db0db87dbfee.png)

**Comment by smolendawid** [comment]
Yes, I can see it. I misled you, the disk is mounted and my eqMac.app is in the  /Applications folder. After clicking eqMac.app nothing happens.

**Comment by victorsoh619** [comment]
> Yes, I can see it. I misled you, the disk is mounted and my eqMac.app is in the /Applications folder. After clicking eqMac.app nothing happens.

I'm facing this problem too.

**Comment by ThomasCh666** [comment]
Same problem with me, it stopped working when I upgraded to Catalina. ANy chance we can give you some logs to help? Thx for the very nice app anyway.

**Comment by nodeful** [comment]
I have just released a new version (v0.0.1) of the app. Please download from the website or GitHub and test. This issue might be related to #14 and/or #16

**Comment by eric-hu** [comment]
If others are still running into this issue, these are the symptoms I saw before too. It's been a month since I installed, but I was able to confirm this matched #14 by launching `eqmac` from the command line and seeing the same error logging as in that issue.

I was able to get the app working by allowing eqmac permissions in `System Preferences` => `Security & Privacy` => `General`. I had to reboot _twice_ after doing that to get things working.

**Comment by nodeful** [comment]
Hey everyone 👋 
I will be open sourcing the new version of eqMac, therefore, I am closing this issue and repository as I will be transfering [nodeful/eqMac2](https://github.com/nodeful/eqMac2) repo to [bitgapp/eqMac](https://github.com/bitgapp/eqMac)
Please bear with me for the next couple of days as I setup that new repo. The eqMac2 code will be living in the [old-eqMac2](https://github.com/bitgapp/eqMac/tree/old-eqmac2) branch if anyone is interested in referencing it for whatever reason.
Please try out the new version and report bugs to the new repo.
Thanks for using eqMac.
